### PR TITLE
String changes for enterprise page [Fixes #7375]

### LINKED ISSUE #7375: Minor string changes to new /enterprise page
## Description

Tyler (@skrelnick23) has asked for some minor changes to the /firefox/enterprise page. This page is in en-US only, so we can make string changes with abandon. 

Specs:

- [x] Strike two parts of the headline and subhead
- [x] Add an MSI installer line to the list of installer links
- [x] Make all installer links point at the new /firefox/all/ page

Mockup with specs:

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] List of changes is coded & launched

### PR DESCRIPTION
**Pull request by Errolyn**
## Description
-Removed "Quantum" from title
-Removed "Now with Windows Group Policy support." from description
-Updated download links to point to "/firefox/all/"
## Issue / Bugzilla link
#7375

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -58,4 +58,4 @@ package-lock.json
 static_build
 static_final
 static
-npm-debug.log
+npm-debug.log
```
Comment (amychurchwell) [inline .gitignore]: `.gitignore` needs a blank line at the end of the file :)

File: bedrock/firefox/templates/firefox/enterprise/index.html [hunk 2]
```diff
@@ -72,7 +72,11 @@ <h3>{{ _('Firefox Rapid Release') }}</h3>
     </div>
     <div class="c-links">
       <h3>{{ _('Firefox Extended Support Release (ESR)') }}</h3>
-      <a class="qa-download-esr" href="{{ firefox_url('desktop', 'all', 'esr') }}">{{ _('Download ESR') }}</a>
+      <a class="qa-download-esr" href="{{ firefox_url('desktop', 'all') }}">{{ _('Download ESR') }}</a>
+    </div>
+    <div class="c-links">
+      <h3>{{ _('Firefox MSI Installers') }}</h3>
+      <a class="qa-download-msi" href="{{ firefox_url('desktop', 'all') }}">{{ _('Download MSI') }}</a>
```
Comment (amychurchwell) [inline bedrock/firefox/templates/firefox/enterprise/index.html]: 👏 love that you included `qa-download-msi`! One quick note is that we will need to add a test for this link in `tests/pages/firefox/enterprise.py`:
https://github.com/mozilla/bedrock/blob/af951623c9b468b16ebf9711d7bccece3276c13f/tests/pages/firefox/enterprise.py#L16
We can essentially copy the `_download_esr_locator` line for msi. 😄

Comment (Errolyn) [reply bedrock/firefox/templates/firefox/enterprise/index.html]: 👍  added

File: .gitignore (hunk 1 above)
Comment (Errolyn) [reply .gitignore]: fixed

**Pull request merged**